code/modules/swarmers/swarmer_objs.dm

```diff
@@ -127,10 +127,9 @@
 
 /obj/structure/swarmer/blockade/CanAllowThrough(atom/movable/O)
```

Comment:
please set ``density = true`` on ``/obj/structure/swarmer/blockade`` instead of doing this, this change will not follow the intended functionality of the ``CanAllowThrough`` proc as it discards the base proc return value (which is ``!density``)

Reply:
![image](https://user-images.githubusercontent.com/10467687/90601069-f731c200-e1cd-11ea-8733-fcaa69b46c77.png)